pandas-stubs/io/formats/style.pyi

```diff
@@ -1,97 +1,267 @@
-    def set_table_styles(self, table_styles): ...
-    def set_na_rep(self, na_rep: str) -> Styler: ...
-    def hide_index(self): ...
-    def hide_columns(self, subset): ...
-    def highlight_null(self, null_color: str = ...): ...
+    def apply(
+        self,
+        func: Callable,
+        axis: Axis | None = ...,
+        subset: Subset | None = ...,
+        **kwargs,
+    ) -> Styler: ...
+    def apply_index(
+        self,
+        func: Callable,
+        axis: int | str = ...,
+        level: Level | list[Level] | None = ...,
+        **kwargs,
+    ) -> Styler: ...
+    def applymap_index(
+        self,
+        func: Callable,
+        axis: int | str = ...,
+        level: Level | list[Level] | None = ...,
+        **kwargs,
+    ) -> Styler: ...
+    def applymap(
+        self, func: Callable, subset: Subset | None = ..., **kwargs
+    ) -> Styler: ...
+    # def where(self, cond: Callable, value: str, other: str | None = ..., subset: Subset | None = ..., **kwargs) -> Styler: ...
+    # def set_precision(self, precision: int) -> StylerRenderer: ...
+    def set_table_attributes(self, attributes: str) -> Styler: ...
+    def export(self) -> dict[str, Any]: ...
+    def use(self, styles: dict[str, Any]) -> Styler: ...
+    uuid: Any  # Incomplete
+    def set_uuid(self, uuid: str) -> Styler: ...
+    caption: Any  # Incomplete
+    # def set_caption(self, caption: str | tuple) -> Styler: ...
+    def set_sticky(
+        self,
+        axis: Axis = ...,
+        pixel_size: int | None = ...,
+        levels: Level | list[Level] | None = ...,
+    ) -> Styler: ...
+    def set_table_styles(
+        self,
+        table_styles: dict[Any, CSSStyles] | CSSStyles | None = ...,
+        axis: int = ...,
+        overwrite: bool = ...,
+        css_class_names: dict[str, str] | None = ...,
+    ) -> Styler: ...
+    # def set_na_rep(self, na_rep: str) -> StylerRenderer: ...
+    # def hide_index(self, subset: Subset | None = ..., level: Level | list[Level] | None = ..., names: bool = ...,) -> Styler: ...
+    def hide_columns(
+        self,
+        subset: Subset | None = ...,
+        level: Level | list[Level] | None = ...,
+        names: bool = ...,
+    ) -> Styler: ...
+    def hide(
+        self,
+        subset: Subset | None = ...,
+        axis: Axis = ...,
+        level: Level | list[Level] | None = ...,
+        names: bool = ...,
+    ) -> Styler: ...
     def background_gradient(
         self,
-        cmap=...,
-        low=...,
-        high=...,
-        axis=...,
-        subset=...,
-        text_color_threshold=...,
+        cmap: str = ...,
+        low: float = ...,
+        high: float = ...,
+        axis: Axis | None = ...,
+        subset: Subset | None = ...,
+        text_color_threshold: float = ...,
         vmin: float | None = ...,
         vmax: float | None = ...,
-    ): ...
-    def set_properties(self, subset=..., **kwargs): ...
+        gmap: Sequence | None = ...,
```

Comment:
Shouldn't use `Sequence` as that would accept a plain string.  Maybe `list | tuple | np.ndarray | Series | DataFrame | None` ??